### ISSUE TITLE
Same named assets display on wrong seller account

### ISSUE THREAD

**Issue by MattL-V** [hugoattal/OrbitalMarket#64]
https://orbital-market.com/search?searchText=author%3A%22Kyrylo+Sibiriakov%22

Has a "Favela" asset from Kitbash3D that even redirects to its proper FAB site.
However, on Orbital it is parsed as Kyrylo asset, probably because he has another "Favela" named asset.
Probably that's why on Orbital's Kitbash3D seller page, their "Favela" asset is missing.

**Comment by hugoattal** [comment]
Yes, I'm aware of this... Since Fab removed the slug, I'll have to create non-conflicting ones. I'll do it when I find a little time.
